config/locales/en/course/user_registrations.yml

```diff
@@ -11,5 +11,7 @@ en:
         new_enrol_request_button: 'Request to enrol'
       create:
         invalid_code: 'You have entered an invalid invitation code.'
+        code_taken: 'The invitation code is used by another account, please login that account instead.'
```

Comment:
`The provided invitation code has been claimed by another account, please use that account to log in instead.`